### DISCUSSION TITLE
How to batch insert images from note attachments into the note body？

### DISCUSSION THREAD

**Discussion by skyrim663** [TriliumNext/Trilium#3248] [Q&A]
![屏幕截图 2022-10-27 195720](https://user-images.githubusercontent.com/68507171/198278419-f55bea88-6acb-4a84-9660-efa1c57d52e3.png)
How do I import the entire image file imported through the import file into the body of my notes?
Currently I can only drag them one by one into the body of the note to display them properly

![屏幕截图 2022-10-27 200237](https://user-images.githubusercontent.com/68507171/198279207-93a125cd-3e4b-4977-b942-b4901a194528.png)

If I CTRL+A select all the images and drag them into the note, it will only show the file name but not the image preview, like above.

**Comment by zadam** [comment]
There's no batch support for this and you'll have to do it one by one.
